geomesa-tools/README.md

```diff
@@ -126,19 +130,12 @@ Due to licensing restrictions, a number of necessary dependencies required for r
         <artifactId>jai_imageio</artifactId>
         <version>1.1</version>
     </dependency>
-    <dependency>
-        <groupId>java3d</groupId>
-        <artifactId>vecmath</artifactId>
-        <version>1.3.2</version>
-    </dependency>
 
 
-To install, you can either locate the jar files or run the two following included scripts which will attempt to wget and install the jars.
+To install, you can either locate the jar files or run the following included script which will attempt to wget and install the jars.
```

Comment:
I agree we should standardize on "JAR", and we are using "JAR" exclusively in the RST docs. The READMEs, however, are more inconsistent on this. I think I used I followed the existing usage in an attempt to tread lightly, but I will fix it.